devtools/common/src/main/java/io/quarkus/cli/commands/file/BuildFile.java

```diff
@@ -23,7 +23,7 @@
 
     protected final static Printer PRINTER = new Printer();
 
-    private final ProjectWriter writer;
+    protected final ProjectWriter writer;
```

Comment:
there is a `getWriter()` method - use that instead.

Reply:
Didn't see it, good catch!